sigstore/_internal/trustroot.py

```diff
@@ -98,13 +98,7 @@ def staging(cls, offline: bool = False) -> "TrustedRoot":
     @staticmethod
     def _get_tlog_keys(tlogs: list[TransparencyLogInstance]) -> Iterable[bytes]:
         """Return public key contents given transparency log instances."""
-
-        for key in tlogs:
-            if not _is_timerange_valid(key.public_key.valid_for, allow_expired=False):
-                continue
-            key_bytes = key.public_key.raw_bytes
-            if key_bytes:
-                yield key_bytes
+        return [key.public_key.raw_bytes for key in tlogs]
 
```

Comment:
are we sure this change is correct for rekor keys?

Reply:
What was the reason for removing `_is_timerange_valid`?

Reply:
`_is_timerange_valid` was the way to get only the active keys, but we should use inactive keys as well. Could we keep the generator allowing expired?

Reply:
Whoops, I missed this comment during review. ++ to @haydentherapper's question about the timerange check.

CC @javanlacerda